desktop/app.js

```diff
@@ -83,6 +84,11 @@ module.exports = function main() {
       // when you should delete the corresponding element.
       mainWindow = null;
     });
+
+    // wait until window is presentable
+    mainWindow.once('ready-to-show', function() {
+        mainWindow.show();
+    });
```

Comment:
do we need to be passing the `this` context here? if not we can simplify

```js
mainWindow.once('ready-to-show', mainWindow.show);
```

Reply:
That would work just as well. Let me know if you want me to update the pull request.

Reply:
@secretfork cool if you don't mind updating to that one last time I will merge this!

Reply:
@roundhill pushed the change dmsnell suggested (i assume that is what you meant), so feel free to review and merge :+1: